### PR TITLE
powerpc/ppc64: unmask media-libs/libva

### PR DESCRIPTION
**Pull request by a17r**

### CODE REVIEW

File: profiles/arch/powerpc/ppc64/64le/package.mask [hunk 1]
```diff
@@ -73,25 +60,10 @@
...
--media-plugins/gst-plugins-vaapi
-
 # Georgy Yakovlev <gyakovlev@gentoo.org> (2019-08-14)
 # works fine on little-endian
 -x11-drivers/xf86-video-amdgpu
 
-# Georgy Yakovlev <gyakovlev@gentoo.org> (2019-02-04)
-# Temurin ppc64 build is little-endian only
-# so we can use bins.
--dev-java/openjdk-bin
```
Comment (arthurzam) [inline profiles/arch/powerpc/ppc64/64le/package.mask:93]: why? We unmask openjdk-bin since we have official binary for this ppc64le?

Comment (a17r) [reply profiles/arch/powerpc/ppc64/64le/package.mask:93]: this was reported as UnmatchedProfilePackageUnmask, so I assumed it was free to cleanup.

File: profiles/arch/powerpc/ppc64/64le/package.mask
Comment (arthurzam) [inline profiles/arch/powerpc/ppc64/64le/package.mask:1]: rechecked all the stuff, and all those unmasks are "anti mask" for ppc64be, but for those we cleaned up the arches well, so no need for those unmasks.

Thank you.

File: profiles/arch/powerpc/ppc64/package.mask
Comment (arthurzam) [inline profiles/arch/powerpc/ppc64/package.mask:1]: please for now move those masks you drop here to `profiles/arch/powerpc/ppc64/64ul/package.mask` (meaning for big endian ppc64) until further verification of libva working on big endian.

ppc64le is much stabler and usable than ppc64be.

File: profiles/arch/powerpc/ppc64/package.use.mask
Comment (arthurzam) [inline profiles/arch/powerpc/ppc64/package.use.mask:1]: similary, please move those masks to the big endian profile: `profiles/arch/powerpc/ppc64/64ul/package.use.mask` until further investigation.

**Comment by arthurzam** [comment]
Pinged powerpc users (in #gentoo-powerpc IRC) if they know how well libva works on big endian ppc64.

File: profiles/arch/powerpc/ppc64/package.mask
Comment (a17r) [reply profiles/arch/powerpc/ppc64/package.mask:1]: it seems they are already masked on 64ul.

**Comment by arthurzam** [comment]
@a17r after some checks with powerpc users, libva somewhat works and somewhat not on powerpc64 big-endian. I think this is good enough to simplify the profiles and just unmask it across ppc64 profiles.

**Comment by sdhoward** [comment]
> we cleaned up the arches well

Prior to this PR, a package was masked in ppc64ul (both endians) and then unmasked in ppc64le. This had the intended effect of masking it only on ppc64 (be).

In this PR, that work was undone. At least one unmask was removed from ppc64le, while the mask remained in ppc64 (be) - https://bugs.gentoo.org/925111#c8

Was the pkgcheck tool reporting [UnmatchedProfilePackageUnmask](https://pkgcore.github.io/pkgcheck/man/pkgcheck.html#unmatchedprofilepackageunmask) incorrectly? If so, please follow up with a bug.

Also you said that even though a package is not working on ppc64 (be), it can be unmasked. The reasoning is hard to follow.

**Comment by thesamesam** [comment]
> Was the pkgcheck tool reporting [UnmatchedProfilePackageUnmask](https://pkgcore.github.io/pkgcheck/man/pkgcheck.html#unmatchedprofilepackageunmask) incorrectly? If so, please follow up with a bug.

I already did at https://github.com/pkgcore/pkgcheck/issues/685 for https://bugs.gentoo.org/918647#c5.

> Also you said that even though a package is not working on ppc64 (be), it can be unmasked. The reasoning is hard to follow.

I covered that in the commit message (86238aee232f1138303d8502ebc608f8bbbaac62).

Anyway, please file a bug for the broken unmasks.

**Comment by thesamesam** [comment]
I'll just revert it now as matoro asked.
